api/src/client/WebserviceLicenseStatusProcessorClient.ts

```diff
@@ -99,7 +101,8 @@ export const determineLicenseStatus = (value: string): LicenseStatus => {
     case "Withdrawn":
       return "WITHDRAWN";
     default:
-      return "UNKNOWN";
+      logWriter.LogError(`Cannot find license status ${value}`);
```

Comment:
Boulder - This message sent to the logger is no consistent, how would I be able to debug this? What information is missing?